src/AbstractTrackingListener.php

```diff
@@ -208,7 +209,7 @@ abstract protected function getFieldValue($meta, $field, $eventAdapter);
      *
      * @return void
      */
-    protected function updateField($object, $eventAdapter, $meta, $field)
+    protected function updateField($object, $eventAdapter, $meta, $field, array $config = [])
```

Comment:
This change breaks BC.

Reply:
Right, you're supporting PHP 7.2 ... I'll remove the property type.

Reply:
I just checked, type-hinting arrays is available in the supported PHP version. The parameter does have a default value, so unless someone passes anything other than an array this won't change a thing. Are am I missing something?